### PR TITLE
Proto File Compiled To CommonJS for better Webpack support

### PR DESCRIPTION
**Pull request by DeekyJay**

### CODE REVIEW

File: lib/robot/packets.js [hunk 1]
```diff
@@ -1,11 +1,6 @@
 import { FatalCodingError, UnknownPacketError } from '../errors';
-import Protobuf from 'protobufjs';
 import varint from 'varint';
-import path from 'path';
-
-const builder = Protobuf.loadProtoFile(path.join(__dirname, 'tetris.proto'));
-const Packets = builder.build('tetris');
-
+const Packets = require('./tetris').tetris;
```
Comment (connor4312) [inline lib/robot/packets.js]: `import { tetris as Packets } from './tetris';`

Comment (Leviathan5) [inline lib/robot/packets.js]: Oh yes of course. Silly me!

File: package.json [hunk 2]
```diff
@@ -32,7 +32,6 @@
   "dependencies": {
     "lodash.assignin": "^4.0.2",
     "lodash.isplainobject": "^4.0.1",
-    "pbjs": "0.0.3",
```
Comment (ProbablePrime) [inline package.json]: this is still used in the npm scripts, should it be in devdependancies? Couldn't see it there :S?

Comment (ProbablePrime) [inline package.json]: Gotcha :)

**Comment by SimonSchick** [comment]
I don't believe this file should be on git but only on NPM when published.

**Comment by Leviathan5** [comment]
Removing the file causes eslint to error as it is unable to resolve path of the module in packets.js.

This is the only reason I left it in the repo. Let me know if you have any suggestions to workaround this so we can remove the file. :+1:

**Comment by SimonSchick** [comment]
Well you could add a `// eslint-disable-next-line: RULENAME` before the require.

**Comment by Leviathan5** [comment]
Was not sure if eslint ignoring was cool right inside that module, but I've made the change.
I also left the `lib/robot/tetris.js` inside the `.eslintignore` as the proto file gets compiled in the `lib/robot` directory. An alternative is just compiling the proto right into the dist and removing that ignore. Let me know what you think. :+1:

**Comment by SimonSchick** [comment]
That's fine.

**Comment by ProbablePrime** [comment]
With the onset of interactive 2 I am going to close this, @DeekyJay @SimonSchick

**Pull request closed without merging**